tests/test_mp_full.py

```diff
@@ -0,0 +1,95 @@
+"""
+multiproc full tests.
+"""
+
+import multiprocessing
+import platform
+import pytest
+import time
+
+import six
+import wandb
+
+
+def train(add_val):
+    time.sleep(1)
+    wandb.log(dict(mystep=1, val=2 + add_val))
+    wandb.log(dict(mystep=2, val=8 + add_val))
+    wandb.log(dict(mystep=3, val=3 + add_val))
+    wandb.log(dict(val2=4 + add_val))
+    wandb.log(dict(val2=1 + add_val))
+    time.sleep(1)
```

Comment:
why do we need these sleeps?

Reply:
not needed, will remove in next. set of mp tests